ios/Classes/codec/AblyFlutterWriter.m

```diff
@@ -27,6 +27,8 @@ + (UInt8) getType:(id)value{
         return messageCodecType;
     }else if([value isKindOfClass:[ARTTokenParams class]]){
         return tokenParamsCodecType;
+    }else if([value isKindOfClass:[ARTPaginatedResult class]]){
```

Comment:
I'm not going to block this pull request landing on this, but I am interested to find out why our linter didn't pick up the fact that there's no space before the `else`, nor after the `if`... nor before the final `{` actually! 🤔 

It should be configured so this fails the build.

Reply:
Lint was fixed in another PR #67  dedicated purely for linting.
Even though it majorly focuses on dart code linting, I've also applied formatting to java source and auto indent to ObjC source.

Reply:
Thanks. Sorry, I should have perhaps looked ahead in order not to add this noise here. 😅